spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -158,28 +158,30 @@ class CometSparkSessionExtensions
           // data source V1
           case scanExec @ FileSourceScanExec(
                 HadoopFsRelation(_, partitionSchema, _, _, _: ParquetFileFormat, _),
-                _: Seq[AttributeReference],
+                attrs: Seq[_],
```

Comment:
I'm not sure if its needed tbh, can be `FileSourceScanExec` be constructed in some other way? I'd say we can just replace it with underscore like other params

Reply:
I'm not sure either. I was just trying to keep my changes semantically equivalent with the current code to avoid risk.

Reply:
@viirya do we really need such kind of type check?

Reply:
It looks like unnecessary.

Reply:
I have now changed these to `Seq[_]`